graphql/src/graphql_schema.ml

```diff
@@ -396,6 +437,14 @@ module Make(Io : IO) = struct
     | _ ->
         invalid_arg "Arguments must be Interface/Union and Object"
 
+  let obj_of_subscription_obj {name; doc; fields} =
+    let fields = List.map
+      (fun (SubscriptionField {name; doc; deprecated; typ; args; resolve}) ->
+        Field { lift = Obj.magic (); name; doc; deprecated; typ; args; resolve = (fun ctx () -> resolve ctx) })
+      fields
+    in
+    { name; doc; abstracts = ref []; fields = lazy fields }
```

Comment:
I've been thinking about how to avoid this, but I don't have a solution ready yet. Maybe it'll have to wait for second iteration on subscriptions.

Reply:
This doesn't feel like a huge problem to me right now.